src/plugins/auto/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -9,6 +9,16 @@
 using namespace ov::test::behavior;
 
 namespace {
+const std::vector<ov::AnyMap> inproperties = {
+    {ov::device::id("UNSUPPORTED_DEVICE_ID_STRING")},
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         OVPropertiesIncorrectTests,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_MULTI),
```

Comment:
```suggestion
                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_MULTI, ov::test::utils::DEVICE_AUTO),
```

Reply:
Seems the original template plugin no the test for auto plugin. This PR is only for moving the auto/batch/multi/hetero plugin tests out from template plugin. So do we need to add it here?
https://github.com/openvinotoolkit/openvino/blob/dc8db45db4b594b94f2089f49245a2acd8934f60/src/plugins/template/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp#L17-L25